apps/web/src/app/_components/combo-box.tsx

```diff
@@ -51,15 +51,18 @@ export default function ComboBox({
   const styleVariant =
     variant === "form"
       ? "flex h-16 w-full rounded-md border-[3px] border-cooper-blue-600 bg-white px-3 py-2 text-xl font-normal ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-0 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
-      : "";
+      : variant === "filtering"
+        ? "w-[340px] h-12 rounded-none border-2 border-l-0 border-t-0 border-[#9A9A9A] text-lg placeholder:opacity-50 focus:ring-0 active:ring-0 lg:rounded-md lg:border-2 py-0"
```

Comment:
Instead of `340px` can we use 340/16 ~ `21rem`

Also for the border let's use `border-[0.75px]` to be consistent with the Figma and all the other borders we use throughout the app.

If possible, would you also be able to make it so that the placeholder text is the same color as the border?